src/Shared/LayoutRenderers/AspNetResponseStatusCodeLayoutRenderer.cs

```diff
@@ -15,7 +15,7 @@ namespace NLog.Web.LayoutRenderers
     /// </code>
     /// </example>
     [LayoutRenderer("aspnet-response-statuscode")]
-    public class AspNetResponseStatusCodeRenderer : AspNetLayoutRendererBase
+    public class AspNetResponseStatusCodeLayoutRenderer : AspNetLayoutRendererBase
```

Comment:
Though it seems tempting, then you can repair past mistakes with major-version-bump (Ex. NLog.Web v6)

Please revert, then we can make an issue for v6 about fixing this.